bundle/CMakeLists.txt

```diff
@@ -127,16 +127,15 @@ endif()
 #---------------------------------
   if(CLONE_RRFSDATA)
      set(RDAS_RRFS_DATA_ROOT "$ENV{RDAS_RRFS_DATA_ROOT}/rrfs-test-data")
-#    set(RDAS_RRFS_DATA_ROOT "/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/rrfs-test-data")
      set(DESTINATION_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
      if(NOT EXISTS "${DESTINATION_DIR}/rrfs-test-data")
         file(MAKE_DIRECTORY "${DESTINATION_DIR}/rrfs-test-data")
      endif()
      if(FV3_DYCORE)
-     file( COPY "${RDAS_RRFS_DATA_ROOT}/rundir-rrfs_fv3jedi_hyb_2022052619" DESTINATION "${DESTINATION_DIR}/rrfs-test-data" )
+     file( COPY "${RDAS_RRFS_DATA_ROOT}/rrfs-data_fv3jedi_2022052619" DESTINATION "${DESTINATION_DIR}/rrfs-test-data" )
```

Comment:
for easy to understand, I would suggest use names like "base_run_dir-rrfs_fv3jedi_hyb_2022052619" or something like that.

Reply:
My thought was that we would have case-agnostic data directories. That way the fv3jedi directory here could also be used for the LETKF ctest. Would including "hyb" here and in `RDASApp/bundle/rrfs-test-data` make it more confusing?